SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -741,7 +751,10 @@ - (void)sdl_requestVideoCapabilities:(SDLVideoCapabilityResponseHandler)response
 
         SDLVideoStreamingCapability *videoCapability = ((SDLGetSystemCapabilityResponse *)response).systemCapability.videoStreamingCapability;
         self.videoStreamingCapability = videoCapability;
-        self.touchManager.scale = self.sdl_scale;
+        const float scale = self.sdl_scale;
+        self.touchManager.scale = scale;
+        [self.carWindow setScale:scale];
+        [self.focusableItemManager setScale:scale];
```

Comment:
Can you format `carWindow` and `focusableItemManager` to set the `scale` value the same way `touchManager`'s `scale` value is set. You don't need to add a `setScale` method; just make the `scale` value public in the header file for `SDLCarWindow` and `SDLFocusableItemLocator`.

Reply:
done